docs/docs/data_classification_policy.rst

```diff
@@ -0,0 +1,93 @@
+.. _llm_as_judge:
+
+.. note::
+
+   To use this tutorial, you need to :ref:`install unitxt <install_unitxt>`.
+
+=====================================
+Sensitive data in unitxt ✨
+=====================================
+
+The section discusses how to properly handle sensitive data in unitxt in order to avoid accidentally sending propriety/confidential data to external services.
```

Comment:
```suggestion
The section discusses how to properly handle sensitive data in unitxt in order to avoid accidentally sending proprietary/confidential data to external services.
```